docs/src/model_setup/forcing_functions.md

```diff
@@ -277,3 +280,88 @@ ContinuousForcing{Nothing} at (Center, Center, Center)
 └── field dependencies: (:T,)
 ```
 
+## `AdvectiveForcing`
+
+`AdvectiveForcing` defines a forcing function that represents advection by
+a separate or "slip" velocity relative to the prognostic model velocity field.
+`AdvectiveForcing` is implemented with native Oceananigans advection operators,
+which means that tracers advected by the "flux form" advection term
+``∇ ⋅ u⃗ˢˡⁱᵖ c``. Caution is advised when ``u⃗ˢˡⁱᵖ`` is not divergence free.
+
+As an example, consider a model for sediment settling at a constant rate:
+
+```jldoctest
+using Oceananigans
+
+r_sediment = 1e-4 # "Fine sand"
+ρ_sediment = 1200 # kg m⁻³
+ρ_ocean = 1026 # kg m⁻³
+Δb = 9.81 * (ρ_ocean - ρ_sediment) / ρ_ocean
+ν_molecular = 1.05e-6
+w_sediment = 2/9 * Δb / ν_molecular * r_sediment^2
+
+sinking = AdvectiveForcing(UpwindBiasedFifthOrder(), w=w_sediment)
+
+# output
+AdvectiveForcing with the UpwindBiasedFifthOrder scheme:
+├── u: ZeroField{Int64}
+├── v: ZeroField{Int64}
+└── w: ConstantField(-0.00352102)
+```
+
+The first argument to `AdvectiveForcing` is the advection scheme (here `WENO5()`).
+The three keyword arguments specify the `u`, `v`, and `w` components of the separate
+slip velocity field. The default for each `u, v, w` is `ZeroField`.
+
+Next we consider a dynamically-evolving slip velocity. For this we use `ZFaceField`
+with appropriate boundary conditions as our slip velocity:
+
+```jldoctest sinking
+using Oceananigans
+using Oceananigans.BoundaryConditions: ImpenetrableBoundaryCondition
+
+grid = RectilinearGrid(size=(32, 32, 32), x=(-10, 10), y=(-10, 10), z=(-4, 4),
+                       topology=(Periodic, Periodic, Bounded))
+
+no_penetration = ImpenetrableBoundaryCondition()
+slip_bcs = FieldBoundaryConditions(grid, (Center, Center, Face),
+                                   top=no_penetration, bottom=no_penetration)
+
+w_slip = ZFaceField(grid, boundary_conditions=slip_bcs)
+sinking = AdvectiveForcing(WENO5(; grid), w=w_slip)
+
+# output
+AdvectiveForcing with the WENO5 scheme:
```

Comment:
If the idea is to be consistent, we could use `UpwindBiasedFifthOrder()` here as well, but if the idea is to show that users can choose other schemes, we can keep as it is.

Reply:
The idea is to show we can use different schemes.